### PR TITLE
Add VictoriaMetrics

### PR DESCRIPTION
**Pull request by sh-cho**
victoriametrics

**Issue:** X

**GitHub Stars:** GitHub Repo stars

https://github.com/VictoriaMetrics/VictoriaMetrics

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -13829,6 +13829,17 @@
...
         },
+        {
+            "title": "VictoriaMetrics",
+            "aliases": {
+                "aka": [
+                    "VM"
+                ]
+            },
+            "hex": "110F0F",
+            "source": "https://github.com/VictoriaMetrics/VictoriaMetrics/blob/26cee364991ebec76cf3ebcf0b28beb66e0ff53e/VM_logo.zip",
```
Comment (adamrusted) [inline _data/simple-icons.json:13840]:
```suggestion
            "source": "https://github.com/VictoriaMetrics/VictoriaMetrics/blob/24d61bf19374b42ef9839c13c7d35ce8888170e0/docs/assets/images/vm_logo.svg",
```

**Review by adamrusted — CHANGES_REQUESTED** [review]
I'd probably update the source for this one, and maybe use the purple/orange from the homepage as a brand colour?
When you update the source, probably best re-generating the icon also.

Would you be able to have another look at this one @sh-cho? 🙏

File: _data/simple-icons.json (hunk 1 above)
Comment (sh-cho) [reply _data/simple-icons.json:13840]: Ok, I'll update it :)

Purple(`#621773`) looks good

**Comment by adamrusted** [comment]
And we're back to the linter being awfully specific.
Apologies @sh-cho - could you have another look at this one? If you pull the latest version, the `npm run svglint` command _should_ actually work locally...

**Comment by sh-cho** [comment]
@adamrusted ok :)

**Comment by sh-cho** [comment]
Test is failed since GitHub API is not working well... (https://www.githubstatus.com/incidents/c8zjb9351hlc)

File: icons/victoriametrics.svg
[the remark this answers is not in the document]
Comment (sh-cho) [reply icons/victoriametrics.svg:1]: Ok I'll try later :)

**Comment by PeterShaggyNoble** [comment]
Are you still working on this one, @sh-cho?

**Comment by sh-cho** [comment]
@PeterShaggyNoble Sorry I forgot 🥲
I updated. can you check?

**Pull request merged**